content/blog/2020-12-19-LLVM-IR-to-assembly.md

```diff
@@ -0,0 +1,76 @@
++++
+title = "Modifying LLVM’s IR to Assembly on a Existing ISA with Different Number of Registers."
+[extra]
+bio = """
+   Socrates Wong is a second year PhD student at Cornell who interest lies in Computer Archicture.
+"""
+[[extra.authors]]
+name = "Socrates Wong"
++++
+
+
+Most computer architectures have a fixed number of registers, and such number is often fixed and determined arbitrarily.  Originally when 8086 was released (the original x86), there were 8 registers' and most of them are specialized. However in most recent iterations of x86-64 it has total of 40, including 16 general purpose registers, 8 MMX registers, and 16 SSE registers.  As the registers area overhead in modern out of order processors (O-3) are relatively small, it is often common practice to incorporate more registers than what is absolutely minimum to complete the task.   Many advanced topics in computing architecture such as simultaneous multithreading, Tomasulo's algorithm, and speculative execution depend on the existence of having more physical registers than architectural registers.  
```

Comment:
I think you may want two blank lines after this so it renders as a separate paragraph?